src/defaultTo.js

```diff
@@ -2,8 +2,8 @@ var _curry2 = require('./internal/_curry2');
 
 
 /**
- * Returns the second argument if it is not null or undefined. If it is null
- * or undefined, the first (default) argument is returned.
+ * Returns the second argument if it is not `null`, `NaN` or `undefined`
+ * otherwise the first arguments is returned.
```

Comment:
I would list the "invalid" values in one of the following orders:

  - `[null, undefined, NaN]`
  - `[NaN, null, undefined]`

It seems nice to keep `null` and `undefined` together, since we often say "null or undefined" (and we have a function which considers both values "nil").

Also, there's a typo on the line above:

```diff
- * otherwise the first arguments is returned.
+ * otherwise the first argument is returned.
```